tests/unittests/config/test_schema.py

```diff
@@ -90,6 +95,65 @@ def get_module_variable(var_name) -> dict:
     return schemas
 
 
+class TestVersionedSchemas:
+    def _relative_ref_to_local_file_path(self, source_schema):
+        """Replace known relative ref URLs with full file path."""
+        # jsonschema 2.6.0 doesn't support relative URLs in $refs (bionic)
+        full_path_schema = deepcopy(source_schema)
+        relative_ref = full_path_schema["oneOf"][0]["allOf"][1]["$ref"]
+        full_local_filepath = get_schema_dir() + relative_ref[1:]
+        file_ref = f"file://{full_local_filepath}"
+        full_path_schema["oneOf"][0]["allOf"][1]["$ref"] = file_ref
+        return full_path_schema
+
+    @pytest.mark.parametrize(
+        "schema,error_msg",
+        (
+            ({}, None),
+            ({"version": "v1"}, None),
```

Comment:
Could we also tests version == 22.2 please?
```suggestion
            ({"version": "v1"}, None),
            ({"version": "22.2"}, None),
```